setup.py

```diff
@@ -187,9 +187,22 @@ def _run_vm(self, kernel_dir):
 
         import vmtest.vm
 
-        command = fr"""cd {shlex.quote(os.getcwd())} &&
-	DRGN_RUN_LINUX_HELPER_TESTS=1 {shlex.quote(sys.executable)} -Bm \
-		unittest discover -t . -s tests/helpers/linux {"-v" if self.verbose else ""}"""
+        command = fr"""
+set -e
+
+cd {shlex.quote(os.getcwd())}
+if "$BUSYBOX" [ -e /proc/vmcore ]; then
+    "$PYTHON" -Bm unittest discover -t . -s tests/linux_kernel/vmcore {"-v" if self.verbose else ""}
+else
+    DRGN_RUN_LINUX_HELPER_TESTS=1 "$PYTHON" -Bm \
+        unittest discover -t . -s tests/helpers/linux {"-v" if self.verbose else ""}
+    if "$BUSYBOX" [ -n "$crashkernel" ]; then
```

Comment:
I'd leave off this conditional; I'd rather that we get test failures if `crashkernel` wasn't set properly.

Reply:
This _always_ fails for kernel version `5.16.0-rc8` though (`crashkernel` seems to never be set properly for that version). You can see this both in the relevant [GitHub actions logs](https://github.com/osandov/drgn/runs/4742796728?check_suite_focus=true#step:7:742), and I'm also able to reproduce it locally.

Reply:
Ah, see torvalds/linux@71d2bcec2d4d69ff109c497e6611d6c53c8926d4, which was merged in v5.16-rc7. Before that commit, the `crashkernel` command line parameter wasn't recognized as a command line parameter, so the kernel passed it to the environment of `init`. It seems like this was by accident, so we shouldn't have been looking for it anyways. The correct thing is still to remove this check.